packages/third-parties/package.json

```diff
@@ -16,19 +16,19 @@
   "scripts": {
     "build": "rm -rf dist && tsc -d -p tsconfig.json",
     "prepublishOnly": "cd ../../ && turbo run build",
-    "updateThirdParties": "rm -rf src/**/index.tsx && node scripts/update-third-parties",
+    "update-third-parties": "rm -rf src/**/index.tsx && node scripts/update-third-parties",
     "dev": "tsc -d -w -p tsconfig.json",
     "typescript": "tsec --noEmit -p tsconfig.json"
   },
   "dependencies": {
+    "next": "^13.0.0",
     "third-party-capital": "1.0.17"
   },
   "devDependencies": {
     "outdent": "0.8.0",
```

Comment:
```suggestion
    "next": "13.4.14",
    "outdent": "0.8.0",
```

Reply:
I think `workspace:*` will be a better option than `13.4.14` as we won't have to manually replace the value whenever a new version of `next` is released.

Reply:
Lerna takes care of this for us